tailwind-showcases/src/z-index-helper.js

```diff
@@ -5,7 +5,7 @@ export const getZIndexHtml = (theme) => {
   const cells = classes
     .map(
       (cls, i) => /*html*/ `
-    <div class="${cls}"
+    <div class="${cls} not-prose"
```

Comment:
can we solve it in a better manner by using ShadowDOM and preventing leaks for typography styles into it, wdyt?

Reply:
The `not-prose` use is a "code smell", we should avoid it everywhere